### DISCUSSION TITLE
Faculty benefits expired without my knowledge, and now I have to wait up to a month to get re-approved?

### DISCUSSION THREAD

**Discussion by JarrettBillingsley** [community/community#24002] [New to GitHub]
<p>Sorry to add another complain-thread when it seems like y’all are already swamped with questions about this. I’ve been using GH pages for years to host my course materials (see <a href="https://jarrettbillingsley.github.io/teaching/classes/cs0447/" class="inline-onebox" rel="noopener nofollow ugc">CS 0447 (term 2214)</a>)</p>
<p>Apparently my faculty benefits expired at the end of February but I got no notification and no request to renew. It was only when my pushes stopped updating my GH pages a couple days ago that I realized what had happened.</p>
<p>I’ve already requested benefits, but it says I might have to wait until the end of March for my request to be approved. This is pretty disruptive, being smack in the middle of the term. I will probably have to pay for a month of GH pro to get this sorted, but I’m mostly wondering why there was no indication my support was ending. I’ve checked my spam, and nothing. Would have been nice to get an email a month before it ended so there could be no downtime.</p>

**Comment by JarrettBillingsley** [comment, marked as answer]
<p>Well speak of the devil, I just got approved. Huh!</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I’m sorry you had all that worry - every application is different. Delighted yours was sorted so quickly.</p>
